src/simulation/environment/dentonFluxModel/_UnitTest/test_dentonFluxModel.py

```diff
@@ -97,7 +99,11 @@ def dentonFluxModelTestFunction(show_plots, param1_Kp, param2_LT, param3_z, para
     module.ModelTag = "dentonFluxModule"
     module.kpIndex = param1_Kp
     module.numOutputEnergies = 6
-    module.dataPath = bskPath + '/supportData/DentonGEO/'
+    e_path = get_path(DataFile.DentonGEOData.model_e_array_all)
```

Comment:
add to known issues this is a breaking change

Reply:
Added release note and known issue.